components/pipeline-service/staging/base/pac-app-name.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: replace
+  path: /data/application-name
+  value: RHTAP Staging
```

Comment:
Adding context from our discussions with the Naming/Branding team. Use of the acronym is acceptable here because users in staging will either be Red Hat employees, or select users who we are working with directly.

For production we cannot assume that anyone knows what RHTAP means until we define the acronym explicitly.